boostrap_test.go

```diff
@@ -10,7 +10,7 @@ import (
 func TestDatadog(t *testing.T) {
 	ddCfg := config.DatadogConfig{}
 
-	err := StartDatadog(ddCfg, false, false)
```

Comment:
Some related comments: https://github.com/coopnorge/helloworld/pull/273#discussion_r1226783803

Reply:
I have extended `DatadogParameters`, `DatadogConfig` to include the option to enable the extra profiling.